src/main/resources/xslt/preservica2solr.xsl

```diff
@@ -35,6 +36,24 @@
         <xsl:for-each select="/xip:DeliverableUnit/Metadata/pbc:PBCoreDescriptionDocument">
           <xsl:call-template name="pbc-metadata"/>
         </xsl:for-each>
+
+
+
+        <!-- Manifestations are extracted here. I would like to create a template for this.
+             However, this is quite tricky when using the document() function -->
+        <xsl:if test="$relation != ''">
+          <f:array key="resource_id">
+            <xsl:for-each select="document($relation)/xip:Manifestation/ComponentManifestation/FileRef">
```

Comment:
That was surprisingly direct and usable, seen from the perspective of general XSLT capabilities!